### PR TITLE
test(dbpull): split tests in multiple files

### PR DESCRIPTION
**Pull request by Jolg42**
Next PR will be about https://github.com/prisma/schema-team/issues/407

Note: things might get renamed & rearranged later, this is a clean split.

It should also limit the blast radius of https://github.com/prisma/prisma/issues/8031

### CODE REVIEW

File: packages/get-platform/src/test-utils/jestSnapshotSerializer.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 const path = require('path')
+const dedent = require('strip-indent')
```
Comment (Jolg42) [inline packages/get-platform/src/test-utils/jestSnapshotSerializer.js:2]:
```suggestion
```

File: packages/get-platform/src/test-utils/jestSnapshotSerializer.js [hunk 2]
```diff
@@ -154,6 +155,7 @@ module.exports = {
       normalizeMigrateTimestamps,
       // artificial panic
       normalizeArtificialPanic,
+      dedent,
```
Comment (Jolg42) [inline packages/get-platform/src/test-utils/jestSnapshotSerializer.js:158]:
```suggestion
```

File: packages/migrate/src/__tests__/DbPull/cockroachdb.test.ts [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+// To avoid the loading spinner locally
+process.env.CI = 'true'
+
+const originalEnv = { ...process.env }
+
+describeIf(!process.env.TEST_SKIP_COCKROACHDB)('cockroachdb', () => {
+  if (!process.env.TEST_SKIP_COCKROACHDB && !process.env.TEST_COCKROACH_URI_MIGRATE) {
+    throw new Error('You must set a value for process.env.TEST_COCKROACH_URI_MIGRATE. See TESTING.md')
+  }
```
Comment (Druue) [inline packages/migrate/src/__tests__/DbPull/cockroachdb.test.ts:28]: I think this was how it was before too but it still reads weirdly that we already discriminate on `!process.env.TEST_SKIP_COCKROACHDB` in the `describeIf()` but then do it again in the `if`

Comment (Jolg42) [reply packages/migrate/src/__tests__/DbPull/cockroachdb.test.ts:28]: Oh yes, this is weird but nothing new here. That code does run somehow if the env vars are not set 🙈 (which in theory should not happen)

**Comment by Jolg42** [comment]
I'll merge as this is a "clean" split passing CI

**Pull request merged**